deployment/tapp-controller.yaml

```diff
@@ -43,7 +43,26 @@ spec:
             - "--v=3"
             - "--register-admission=true"
           imagePullPolicy: "Always"
+          ports:
+          - containerPort: 8443
+            protocol: "TCP"
+            name: "https"
           resources:
             requests:
-              "cpu": "1"
+              "cpu": "100m"
```

Comment:
100m is too small

Reply:
So it can startup on my tiny experiment cluster